src/main/java/com/asana/errors/AsanaError.java

```diff
@@ -21,6 +21,11 @@ public AsanaError(String message, int status, HttpResponseException exception) {
     public static AsanaError mapException(HttpResponseException exception) throws AsanaError {
         switch (exception.getStatusCode()) {
             case ForbiddenError.STATUS:
+                String constructedErrorMsg = constructMessage("", exception);
+                String premiumOnlyStr = "not available for free";
+                if (constructedErrorMsg != null && constructedErrorMsg.contains(premiumOnlyStr)) {
```

Comment:
When would `constructedErrorMsg` ever be `null`? If an exception is thrown inside `constructMessage` then the initial string `""` is returned. However, this sort of defensive programming is on-brand for Java.

Reply:
Yeah I think this was mostly quick paranoia, but you're right.